content/general-docs/introduction.adoc

```diff
@@ -3,11 +3,81 @@
 
 ifndef::include-only-once[]
 :root-path: ../
+:imagesdir: {root-path}/_images
+:includedir: {root-path}/_images
 include::{root-path}_config.adoc[]
 endif::[]
 
 == Overview
-Overview of {THIS_STANDARD}.
+
+=== Motivation
+
+Accurate sensor simulation that can replicate physical phenomena is necessary for safety evaluation of AD/ADAS in virtual space. 
+
+However, previous simulations were heavily limited by computational power and available memory, and while they appeared visually believable, they were not physically accurate.
+As a result, previous simulations were primarily used during the early stages of development, where precise physical modeling was not necessarily required. 
+
+With increased computing power and streamlined simulation techniques, virtual testing plays a more powerful role at all stages of system development.
+This correlates with the growing need to accurately represent the real world in simulations, so-called digital twins, especially as the need for physical sensor simulation increases. 
+
+One fundamental step to describing the real world virtually is a full physical description of all objects, including their material properties and their 3D geometries.
+
+It is extremely important to standardize the representation of data and attributes, such as scenarios, road data, and asset data, necessary for AD/ADAS simulation in order to promote the circulation of these data and to obtain consistent results among different systems. 
+
+Standardization activities have been conducted for scenarios and road data using OpenSCENARIO, OpenDRIVE, OpenCRG, and other standards.
+ASAM OpenMATERIAL closes the gap to advance standardization by providing standardizes definitions of material properties and 3D geometry.
```

Comment:
We have to think if we want to stick with this name, and if so, write somewhere that in this document that we will use ASAM OpenMATERIAL in short, instead of ASAM OpenMATERIAL & 3D model structure

Reply:
The question is, what the official name of the standard will be. ASAM OpenMATERIAL or ASAM OpenMATERIAL & 3D model structure. I would favour simply using ASAM OpenMATERIAL as it is shorter.

Reply:
Agree, we can talk about it this week with the whole group.

Reply:
- Generally a shorter name would be prefered. We could officially name the standard "**ASAM OpenMATERIAL & 3D model structure**" but everybody would just call it "OpenMATERIAL" as a nickname.
- Technically, something like **OpenASSET** or **Open3DASSET** would be more correct, because asset refers to the entire 3D model including its materials. And this definition would enable future additions, like animations etc.
- A workaround could be naming it **OpenMATERIAL+**, **OpenMATERIALx** or **OpenMATERIAL3D** to make clear that there is more to the standard than "just" materials.
- We could have a shorter name that still facilitates both parts, e.g. **OpenMATERIAL-ASSET**, **OpenMATERIAL-GEOMETRY**
- We could just stick with **OpenMATERIAL** and people will learn over time that it also contains geometry.

To be discussed further in the geometry group @LudwigFriedmann. Then we should put it to a vote.

Reply:
Let's move this discussion to the corresponding issue #169 and fix this in another PR.